### ISSUE TITLE
The LUA version of group_send ignores channels capacity

### ISSUE THREAD

**Issue by thomas-riccardi** [django/channels_redis#96]
Previous `group_send` version relied on `send` to check channel capacity, but now that it bypasses `send`, there is no channel capacity check at all, which is dangerous (we got up to 100k messages in a redis queue, partly because of #93).

**Comment by andrewgodwin** [comment]
Agreed this needs to be fixed, thankfully it is in an unreleased version. I will fix it in a few weeks when I am back doing Channels work, or feel free to open a PR.

**Comment by mmcardle** [comment]
I have created a PR that could use other eyes on it, https://github.com/django/channels_redis/pull/101
